openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -153,6 +153,22 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                           {{ interface.wireless.frequency }} GHz
                       </div>
                   </div>
+                  {% if interface.wireless.htmode %}
+                  <div class="form-row">
+                    <label>{% trans 'WiFi Version' %}:</label>
+                    <div class="readonly">
+                    {% if 'NOHT' in interface.wireless.htmode %}
+                        Legacy mode ({{ interface.wireless.htmode }})
+                    {% elif 'HE' in interface.wireless.htmode %}
+                        802.11ax/wifi6 ({{ interface.wireless.htmode }})
+                    {% elif 'VHT' in interface.wireless.htmode %}
+                        802.11ac/wifi5 ({{ interface.wireless.htmode }})
+                    {% elif 'HT' in interface.wireless.htmode %}
+                        802.11n/wifi4 ({{ interface.wireless.htmode }})
+                    {% endif %}
```

Comment:
We need an else clause here to account for unexpected or future values. Eg:

`Other: {{ interface.wireless.htmode }}`.